src/org/jitsi/impl/configuration/ConfigurationServiceImpl.java

```diff
@@ -111,7 +113,8 @@
      * <tt>setProperty()</tt> methods. Still, re-setting one of these properties
      * to <tt>null</tt> would cause for its initial value to be restored.
      */
-    private Map<String, String> defaultProperties = new HashMap<>();
+    private Map<String, String> defaultProperties
+        = new ConcurrentHashMap<>();
```

Comment:
Like immutableDefaultProperties, defaultProperties appears to be modified only during the construction of ConfiguratinoServiceImpl and, consequently, it seems unnecessary to use a ConcurrentHashMap.